### ISSUE TITLE
How can I drag files into neovide dock

### ISSUE THREAD

**Issue by xzbdmw** [neovide/neovide#2555]
**Describe the bug**
with 13.0, I think I can drop files into dock but I can't, downloaded from release page.

**To Reproduce**

https://github.com/neovide/neovide/assets/97848247/e7ac0208-7afe-4d81-a3fd-4309feca8ae8

**Comment by fredizzimo** [comment]
Hm, it looks like we forgot to make the continuation of this, to add CI support
* https://github.com/neovide/neovide/pull/2395

It added some scripts that should be run on the CI and replace the existing `cargo-bundle` solution https://github.com/neovide/neovide/tree/main/macos-builder. 

Cargo bundle unfortunately lacks the features we need, and it looks mostly unmaintained for example, https://github.com/burtonageo/cargo-bundle/pull/147

Could someone finish the CI integration on macOS?

**Comment by falcucci** [comment]
works as expected to me.

https://share.cleanshot.com/RcQhGzMh

**Comment by oschrenk** [comment]
Not expected behabiour depending on your definition. You are dropping onto the application window not the dock!

There are two behaviours in macOS. 
1) is controlling windows, and 
2) is accepting drag/drop

You are showing 1)
The difference is that 1) does not work if the app is not already open and the dock icon is not a drop "zone"

You can see the difference by keeping (rightclick on dock item. Options > Keep in Dock) Neovide.app and Mail.app in Dock but without any windows open.

Now try to drag and drop a file onto neovide. Nothing is happening, while Mail.app will react by darkening it's color and showing the label.

When I drop the log.txt onto the Dock icon of Mail it will open Mail and create a new email (I cut off the video here to protect my email adress)


https://github.com/neovide/neovide/assets/52898/88da3272-6704-40d6-8ac7-d168236b5fa9

**Comment by falcucci** [comment]
That's correct @oschrenk I'll look to see a fix for that, thank you for clarifying it.

**Comment by falcucci** [comment]
A fix has been sent at #2700

In case you want, download the .dmg installer from the latest [artifact](https://github.com/neovide/neovide/actions/runs/9901193975) to test it out.

**Comment by fredizzimo** [comment]
The fix is merged

**Comment by xzbdmw** [comment]
> It added some scripts that should be run on the CI and replace the existing cargo-bundle solution

The section of macos building from source in website still uses cargo-bundle, should that be changed?
